active/NavigationView/NavigationViewTemplateSettings/PaneTitle.md

```diff
@@ -0,0 +1,45 @@
+# Background
+PaneTitle is added to NavigationView in RS4, but it's a String. Since RS5, we introduced PaneHeader(UIElement) to deprecate PaneTitle. If both PaneTitle and PaneHeader are provided by customer, PaneHeader is used.

```

Comment:
We already docprecated PaneTitle, why are we adding new support for it? Does the template have to create elements for both the title and the header?

Reply:
I introduced a new API to support the deprecated PaneTitle just because it's a simplest way to fix the problem.
PaneTitle visibility is controlled by XAML, so it's nearly impossible to maintain a good state if we control it from code. There is a possible solution that we just throw exception if they set title and header at the same time.